lib/tasks/data.rake

```diff
@@ -0,0 +1,65 @@
+namespace :data do
+  require 'open-uri'
+  require 'json'
```

Comment:
Main reason I do it this way is so that you don't load the rake task and then run into dependency issues - happens a lot with test/development group gems / tasks. 

Sticking it inside the namespace tends to avoid being evaluated until you actually execute the task.

Doesn't really matter in this case as json/open-uri are required elsewhere